.storybook/components/Explore/Explore.stories.tsx

```diff
@@ -22,23 +22,41 @@ export default ExploreEventsMeta
 
 type ExploreEventsStory = ComponentStory<typeof ExploreEventsView>
 
-const Stack = createStackNavigator<ExploreEventsScreensParamsList>()
-const screens = createExploreEventsScreens(Stack, async () => [
-  EventMocks.Multiday,
-  EventMocks.NoPlacemarkInfo,
-  EventMocks.PickupBasketball
-])
+const fetchEvents = async () => [
+  EventMocks.MockMultipleAttendeeResponse,
+  EventMocks.MockSingleAttendeeResponse
+]
+
+const storage = AlphaUserStorage.ephemeral()
 
 export const Basic: ExploreEventsStory = () => (
-  <MenuProvider>
-    <TiFQueryClientProvider>
-      <SafeAreaProvider>
-        <NavigationContainer onStateChange={console.log}>
-          <Stack.Navigator screenOptions={{ ...BASE_HEADER_SCREEN_OPTIONS }}>
-            {screens}
-          </Stack.Navigator>
-        </NavigationContainer>
-      </SafeAreaProvider>
-    </TiFQueryClientProvider>
-  </MenuProvider>
+  <AlphaUserSessionProvider storage={storage}>
+    <GestureHandlerRootView>
```

Comment:
Was the GestureHandlerRootView included to help fix issues with the gestures in the app, which were mentioned earlier? 
Those issues being:

- The Journey screen being held when too many gestures are being made
- Not being able to swipe to the previous/next screens, while the map is available

Reply:
Components that require gestures need this component since it acts as a context provider for the gesture handler library (similar to how the bottom sheet library needs a `BottomSheetModalProvider`).